src/util/api/index.js

```diff
@@ -131,3 +131,5 @@ export const getArticleDetailAPI = url =>
 
 export const updateProfile = () => `/api/users/self/update`;
 export const userPreferences = () => `/api/users/preferences`;
+
+export const getOgImage = id => `https://static.like.co/liker-og-image/${id}`;
```

Comment:
Suggest to rename this func for more specific use case
```suggestion
export const getLikerOgImage = id => `https://static.like.co/liker-og-image/${id}`;
```

Reply:
OK! Done~